### PR TITLE
Generating own keys and new addresses based on this.

### PR DESCRIPTION
**Pull request by marpme**

### CODE REVIEW

**Review by SwenVanZanten — APPROVED** [review]
Awesome!! Looks good!

File: package.json [hunk 1]
```diff
@@ -241,6 +245,7 @@
     "styled-components": "^3.3.2",
     "tor-request": "^2.1.1",
     "uuid": "^3.2.1",
-    "verge-node-typescript": "^0.0.16"
+    "verge-node-typescript": "^0.0.16",
+    "xvg-bitcore-lib": "https://github.com/vergecurrency/xvg-bitcore-lib.git"
```
Comment (marpme) [inline package.json:261]: Remove it ...

File: package.json [hunk 2]
```diff
@@ -202,8 +202,12 @@
     "@types/reactstrap": "^5.0.27",
     "@types/uuid": "^3.4.3",
     "@types/webpack-env": "^1.13.6",
+    "bip32": "^1.0.2",
+    "bip39": "^2.5.0",
+    "bitcoinjs-lib": "^4.0.1",
```
Comment (marpme) [inline package.json]: not needed either

**Comment by justinvforvendetta** [comment]
nice, using ledger to create addr! not a bad idea =]

**Comment by marpme** [comment]
Closed until the wallet service (#59) for the client is established.

**Pull request closed without merging**